superset-frontend/src/views/CRUD/welcome/ChartTable.tsx

```diff
@@ -177,13 +179,13 @@ function ChartTable({
       },
     },
   ];
-  if (examples) {
+  if (otherTabData) {
     menuTabs.push({
-      name: 'Examples',
-      label: t('Examples'),
+      name: 'Other',
```

Comment:
nit: shouldn't we use an enum here?

Reply:
same for line 89 actually (`initialFilter === 'Mine'`)

Reply:
+1

Reply:
+1